packages/analyticsdx-template-lint/src/schemas/readiness-schema.json

```diff
@@ -333,7 +333,6 @@
       "minItems": 0,
       "items": {
         "type": "object",
-        "additionalProperties": false,
```

Comment:
Does removing this still cause unknown properties in a filter instance to be marked as bad? e.g. `notvalue` in
```
{ "field": "foo", "operator": "Equal", "notvalue": "bar" }
```
should come back as an unknown property error.

Reply:
great call, my change did cause regression for it.
But if i do not remove this line, i have the following error (some google result is to suggest removing "additionalProperties").  Open for suggestions  
<img width="531" alt="image" src="https://github.com/forcedotcom/analyticsdx-vscode/assets/126512448/e5d66bee-f253-48c8-a0cc-5de88cbd9c03">

Reply:
great call, did cause regression.. but i think i found the right fix now.